### DISCUSSION TITLE
Tracking good answers re ERG Semantics

### DISCUSSION THREAD

**Discussion by emilymbender** [delph-in/docs#78] [Team Posts]
Thanks to all who participated in our session this morning! As mentioned right at the end, I've created a page for us to drop links to places where we found good questions about ERG Semantic representations (and sometimes good answers!) that could be used as inspiration for further entries on the ERG Semantic Documentation pages. Here's the link:

https://github.com/delph-in/docs/wiki/ErgSemantics_Raw_Materials

@arademaker can you add a link to the one you were showing us? Thanks!

**Comment by arademaker** [comment]
This is one example: https://github.com/delph-in/erg/issues/29. I started with a question about why the compound arguments are split into two different labels. @guyemerson ended up teaching me that compound is an under specified preposition. He pointed me to the docs, but his explanation was much more clear to me. Eventually I should have copied it to the docs?

**Comment by emilymbender** [comment]
The ask for right now is just to add that link to this page:

https://github.com/delph-in/docs/wiki/ErgSemantics_Raw_Materials

... so that someone at some later date working on the ESD pages can see it as source material.

**Comment by EricZinda** [comment]
I filled in a bunch from my questions on the forum and places in my code where I've struggled.  Let me know if you want more/different information in there than I provided.

**Comment by arademaker** [comment]
I just did @emilymbender , sorry for miss understanding your initial request.

**Comment by emilymbender** [comment]
Thank you, @arademaker !
